Sources/Commands/SwiftTool.swift

```diff
@@ -74,39 +74,55 @@ private class ToolWorkspaceDelegate: WorkspaceDelegate {
         self.observabilityScope = observabilityScope
     }
 
-    func fetchingWillBegin(repository: String, fetchDetails: RepositoryManager.FetchDetails?) {
+    func willFetchPackage(package: String, fetchDetails: PackageFetchDetails) {
         queue.async {
-            self.outputStream <<< "Fetching \(repository)"
-            if let fetchDetails = fetchDetails {
-                if fetchDetails.fromCache {
-                    self.outputStream <<< " from cache"
-                }
+            self.outputStream <<< "Fetching \(package)"
```

Comment:
For packages that do come from repositories, will there be a way to still see that URL? Based on how this is called it looks as if this is always the string form of the package identity.

Reply:
Sorry, moved my comment from one place in the code right as you were replying.  I see, thanks for clarifying.